src/streamdatasetjson/dataset.py

```diff
@@ -1,34 +1,45 @@
+from typing import Iterable, List, NewType
 import utils
 
+
+Row = NewType("Row", List[str])
+VarMeta = NewType("VarMeta", dict)
```

Comment:
It would be good to have a ItemDescription type from dataset-json [TS schema](https://github.com/cdisc-org/DataExchange-DatasetJson/blob/master/typescript/dataset.ts):

```
ItemDescription = TypedDict(
    "ItemDescription",
    {"OID": str, "name": str, "label": str, "type": str, "length": int},
    total=False
)
```